src/option.cpp

```diff
@@ -508,7 +508,7 @@ std::string Option<T>::str() const
 template<typename T>
 std::string Option<T>::defaultStr() const
 {
-   return(m_default != T{} ? to_string(m_default) : std::string{});
+   return to_string(m_default);
```

Comment:
Yes it does indeed, while I don't think that this is particularly important I thing your proposal of introducing a reset function is better. Still I think that this function should at least be renamed ... as it does return a non default default and not a default default  :dizzy_face:

Reply:
I'm not sure I followed your confusion?

If it helps, it essentially tries to return:
```c++
extern Option
some_name; // = WHATEVER_IS_HERE
```
If you think of it that way, maybe it makes more sense? (The "caveat", of course, is that if the default is a macro or other symbolic constant, you get the *value* of the constant, not the name. Excluding, of course, the well-known enumerations like IARF that are converted back to symbolic form.)

IIRC (it's been a while since #1852), the above exactly describes the intended *semantics* of the function (to wit, it exists specifically for that documentation-related purpose). If you have thoughts for a better name...?